cosmovisor/README.md

```diff
@@ -1,3 +1,30 @@
+# Agoric enhancements to Cosmovisor
```

Comment:
I understand your point.  I like having these changes in README.md because that's what shows up on the [repository's main page](https://github.com/agoric-labs/cosmos-sdk).